### PR TITLE
LPS-183744 Add Account Group On Site Initializer

### PR DESCRIPTION
**Pull request by joaodamasceno2001**

### CODE REVIEW

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joaodamasceno2001** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 1]
```diff
@@ -452,7 +468,9 @@ public void initialize(long groupId) throws InitializationException {
 				siteNavigationMenuItemSettingsBuilder =
 					new SiteNavigationMenuItemSettingsBuilder();
 
+			_invoke(() -> _addAccountGroups(serviceContext));
 			_invoke(() -> _addAccounts(serviceContext));
+			_invoke(() -> _addAccountGroupAssignments(serviceContext));
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:473]:
```suggestion
			
			_invoke(() -> _addAccountGroupAssignments(serviceContext));
```

Comment (nNilton) [reply modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:473]: please add this blank line

**Review by nNilton — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java [hunk 2]
```diff
@@ -357,6 +360,95 @@ public void testInitializeFromFile() throws Exception {
...
+
+		_assertAccountGroupAssignments(accountGroup, 1);
+
+		accountGroup =
+			accountGroupResource.getAccountGroupByExternalReferenceCode(
+				"TESTACCOUNTGROUP3");
+
+		Assert.assertNotNull(accountGroup);
+		Assert.assertEquals("Test Account Group 3", accountGroup.getName());
+		_assertAccountGroupAssignments(accountGroup, 0);
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java:406]:
```suggestion
		
		_assertAccountGroupAssignments(accountGroup, 0);
```

Comment (nNilton) [reply modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test/src/testIntegration/java/com/liferay/site/initializer/extender/internal/test/BundleSiteInitializerTest.java:406]: you need to break line like you did before
same for the others in assertAccountGroups2

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by nNilton** [comment]
ci:forward

**Comment by nNilton** [comment]
waiting for https://github.com/liferay-commerce/liferay-portal/pull/3772

**Pull request closed without merging**